pkg/tuned/dialog.js

```diff
@@ -76,7 +77,7 @@
         }
 
         function update_button() {
-            var tuned = cockpit.dbus('com.redhat.tuned', { superuser: true });
+            var tuned = cockpit.dbus('com.redhat.tuned', { superuser: false });
```

Comment:
`false` is the default (but nitpick). This could use a comment like "reading the current profile works as user". However, do *all* operations we do on com.redhat.tuned are possible as user, in particular changing the current profile? That would seem a bit strange to me, but maybe pcp ships a polkit rule that allows this to normal users.

If `false` breaks, then usually the right thing to do is `superuser: "try"`, see [cockpit.spawn()](https://cockpit-project.org/guide/latest/cockpit-spawn.html).

Reply:
You're absolutely right, you can only change the profile when you have a physical shell or are root. I'll change this and add a test to make sure this doesn't break. This patch tests whether or not you can read a profile as non-root, but not that you *can't* change it.

Superuser try seems good, but I guess we could also just do superuser true. Since the dialog needs admin anyway, and I assume there are few cases where you log into your own system via cockpit.